extensions/dicom-seg/src/components/SegmentationSettings.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import { Range } from '@ohif/ui';
+
+import './SegmentationSettings.css';
+
+const SegmentationSettings = ({ configuration, onBack, onChange }) => {
+  const [state, setState] = useState({ ...configuration });
+
+  useEffect(() => {
+    setState(state => ({ ...state, ...configuration }));
+  }, [configuration]);
```

Comment:
Updates to configuration override previous local. Why this instead of just using all of new configuration? Does it provide different keys at different times?

Reply:
Not necessary to have this one since the parent will have its state updated anyways, thanks.